exploration/default-registry-and-mf1-compatibility.md

```diff
@@ -225,6 +229,41 @@ The functions `:datetime` and `:time` have the following options:
   - `h12`
   - `h23`
   - `h24`
+- `timeZoneName`
+  - `long`
+  - `short`
+  - `shortOffset`
+  - `longOffset`
+  - `shortGeneric`
+  - `longGeneric`
+
+> [!NOTE]
+> The following options do not have default values because they are only to be used
+> as overrides for locale-and-value dependent implementation-defined defaults.
+
+The function `:time` has following options:
+- `hourCycle` (default is locale-specific)
+  - `h11`
+  - `h12`
+  - `h23`
+  - `h24`
+- `timeZoneName`
+  - `long`
+  - `short`
+  - `shortOffset`
+  - `longOffset`
+  - `shortGeneric`
+  - `longGeneric`
+  
```

Comment:
If we're dropping all but style and leaving the locale-ish options out for now, I think these should not be included on `:time`. My feelings aren't too strong here, though.

```suggestion
```

Reply:
Time zone name I could live without here. Hour cycle feels important though: it's a common override on time values. @sffc's point about having a 12/24 hour toggle (`hour12`) also feels like a feature for `:time` specifically.